en/products/vpn/platforms/ios_v2.ftl

```diff
@@ -0,0 +1,15 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/mobile/ios/
+
+vpn-ios-vpn-for-iphone = VPN for { -brand-name-iphone } from { -brand-name-mozilla }
+vpn-ios-looking-for-a = Looking for a VPN for your { -brand-name-iphone }? { -brand-name-mozilla }’s VPN services ensure you are protected online every time you login to your device. Keep reading to learn more about how to keep your { -brand-name-iphone } protected from cyber attacks with { -brand-name-mozilla-vpn } for { -brand-name-iphone }.
+vpn-ios-iphones-are-great = { -brand-name-iphone }s are great at protecting your privacy, but they can’t shield you from all online dangers. That’s why you need { -brand-name-mozilla-vpn }, a service for { -brand-name-iphone } that encrypts your internet traffic and hides your location. In this article, we’ll explain how to use { -brand-name-mozilla-vpn } for { -brand-name-iphone }, what benefits it offers, and how it compares to other VPNs.
```

Comment:
This just doesn't work `{ -brand-name-iphone }s`, it's going to be a mess when localized.